features/resources/lab-result-organization1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "resourceType": "Organization",
+  "name": "KEMRI Clinic123"
```

Comment:
The name here doesn't match the name listed in the questionnaireResponse. How is the search working :|

Reply:
Sorted